ietf/static_src/css/main.scss

```diff
@@ -1,4 +1,22 @@
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+
+$custom-spacers: (
```

Comment:
Everything else in this entry point file is an `@import`. Maybe the spacer stuff should move to a separate file that you import here?

Reply:
moved